.babelrc

```diff
@@ -11,19 +10,28 @@
         },
         "useBuiltIns": "usage"
       }
-    ]
+    ],
+    "@babel/preset-typescript"
   ],
   "plugins": [
-    "styled-components",
+    "babel-plugin-transform-typescript-metadata",
```

Comment:
Where would this be put to use, currently?

Reply:
This plugin enables anything using `Reflect.metadata` to work correctly, which is a number of things – including eg inferring Sequelize datatypes from typescript types. .e.g without it you immediately get

```
Specified type of property 'providerId'
            cannot be automatically resolved to a sequelize data type. Please
            define the data type manually
```

Reply:
Hmm, I still get this error upon `make up` in this branch.

Reply:
I believe I have the magic incantation now